demo/App.jsx

```diff
@@ -1,11 +1,62 @@
-import React from 'react';
+import React, {useState} from 'react';
 import ReactDOM from 'react-dom';
-import {WebexMeetingWidget} from '../src';
+import {Input, Sidebar, SidebarBody, SidebarHeader, SidebarNav, SidebarNavItem} from '@momentum-ui/react';
+
+import WebexMeetingWidgetDemo from './WebexMeetingWidgetDemo';
 
 import './App.css';
 
 export default function App() {
-  return <WebexMeetingWidget accessToken="<YOUR_ACCESS_TOKEN>" meetingDestination="MEETING_DESTINATION" />;
+  const [tokenInput, setTokenInput] = useState('');
+  const [token, setToken] = useState();
+
+  const clearToken = () => {
+    setTokenInput('');
+    setToken();
+  };
+
+  return (
+    <div className="main">
+      <Sidebar withIcons={false}>
+        <SidebarBody>
+          <SidebarNav title="Webex Widgets">
+            <SidebarNavItem title="WebexMeeting" />
+          </SidebarNav>
+        </SidebarBody>
+      </Sidebar>
+      <div className="content">
+        <div className="section">
+          <h3>Webex Widgets Access Token</h3>
+          <h5>Webex Widgets require an access token to identify the current user.</h5>
+          <h5>
+            You can get an access token from <a href="https://developer.webex.com">developer.webex.com</a>
+          </h5>
+          <form className="form">
+            <Input
+              htmlId="token"
+              label="Access Token"
+              name="token"
+              onChange={(event) => setTokenInput(event.target.value)}
+              placeholder="Access Token"
+              readOnly={!!token}
+              type="password"
+              value={tokenInput}
+            />
+            <button onClick={() => setToken(tokenInput)} disabled={token}>
+              Save Token
+            </button>
+            <button onClick={clearToken} disabled={!token}>
+              Clear Token
+            </button>
+          </form>
+        </div>
+        <div className="section">
+          <h2>WebexMeetingWidget</h2>
```

Comment:
Can you use an `h3` to keep the size consistency with the previous section?
```suggestion
          <h3>Webex Meeting Widget</h3>
```